infinispan/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <artifactId>quarkus-langchain4j-infinispan-parent</artifactId>
+    <name>Quarkus LangChain4j - Infinispan embedding store - Parent</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <infinispan.version>15.0.0.Dev07</infinispan.version>
```

Comment:
I understand it's temporary, but let's still put it into the root `pom.xml` along with all other version declarations?